azure-pipelines.yml

```diff
@@ -511,9 +511,30 @@ jobs:
         gcpKeyFile: "gcpkey.json"
         bucketURI: "gs://opentitan-bitstreams/master"
 
-- job: execute_fpga_tests_cw310
-  displayName: CW310 Tests
-  # Execute tests on ChipWhisperer CW310 FPGA board
+- job: execute_test_rom_fpga_tests_cw310
+  displayName: CW310 Test ROM Tests
+  pool: FPGA
+  timeoutInMinutes: 45
+  dependsOn:
+    - chip_earlgrey_cw310
+    - sw_build
+  condition: succeeded( 'chip_earlgrey_cw310', 'sw_build' )
+  steps:
+  - template: ci/checkout-template.yml
+  - template: ci/install-package-dependencies.yml
+  - template: ci/download-artifacts-template.yml
+    parameters:
+      downloadPartialBuildBinFrom:
+        - chip_earlgrey_cw310
+        - sw_build
+  - bash: |
+      set -e
+      . util/build_consts.sh
+      ci/scripts/run-fpga-cw310-tests.sh cw310_test_rom || { res=$?; echo "To reproduce failures locally, follow the instructions at https://docs.opentitan.org/doc/getting_started/setup_fpga/#reproducing-fpga-ci-failures-locally"; exit "${res}"; }
```

Comment:
Would we need vivado for the test rom?

Reply:
I think the answer is no. The Vivado build uses the test ROM by default, so we don't need `updatemem` to splice any bitstreams.